lib/src/json_schema/json_schema.dart

```diff
@@ -1000,6 +1000,11 @@ class JsonSchema {
   static Map<String, SchemaPropertySetter> _accessMapV2019_09 = Map<String, SchemaPropertySetter>()
     ..addAll(_baseAccessMap)
     ..addAll(_accessMapV7)
+    // Unfortunately the spec does not explicitly say to ignore these old keywords,
```

Comment:
could we add a link to the language (or even a quote) in the spec that's closest to what we're referencing here?